src/xcode/ENA/ENA/Source/Models/Async Operation/AsyncOperation.swift

```diff
@@ -0,0 +1,85 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+import Foundation
+
+class AsyncOperation: Operation {
```

Comment:
Why is it called "Async"? I don't see the property "isAsynchronous" set to true.
As far as I know, if its not set, then the operation will be executed synchronous if its called directly or the queue will decide about async or sync.

Reply:
Because it only sets the operation to completed when the task is finished. It allows you to execute async operations serially. Otherwise you could add operations serially to a queue and they would execute in that order but they would not wait for the previous operation to complete first.

Reply:
With this new class we can get rid of all the completion block inceptions we have throughout the app. You could just chain these operations together.